core/server/api/canary/utils/serializers/output/utils/mapper.js

```diff
@@ -52,7 +52,8 @@ const mapPost = (model, frame) => {
     _(metaAttrs).filter((k) => {
         return (!frame.options.columns || (frame.options.columns && frame.options.columns.includes(k)));
     }).each((attr) => {
-        jsonModel[attr] = _.get(jsonModel.posts_meta, attr) || null;
+        const defaultValue = (attr === 'email_only') ? false : null;
+        jsonModel[attr] = _.get(jsonModel.posts_meta, attr) || defaultValue;
```

Comment:
I'm a bit confused as to the purpose of this, is this to stop a `false` value being turned into a `null`? Or is it because we expect that the value _could_ be `undefined` and we want to add a default?

If it's the first, I wonder if something like this is clearer, in that we check if the value exists, rather than if it's truthy, and use a default if it doesn't
```js
const value = _.get(jsonModel.posts_meta, attr);
jsonModel[attr] = _.isUndefined(value) ? null : value;
```

Reply:
Nice one @allouis! It's more of the latter one - having an `undefined` for whole posts_meta object and by extension the `email_only` value. The default for the `email_only` is `false` and this is why we want that `if (undefined) assign "false"` logic. 
I see why it's a bit confusing... the ternary operations are not the most readable thing in the world :sweat_smile: 

I'll refactor this into an if/else with a comment. If the question came out now it will definitely confuse us when we come back to this code in the future!